cardano-cli/test/cardano-cli-golden/files/golden/help/shelley_key_key-from-mnemonic.cli

```diff
@@ -0,0 +1,41 @@
+Usage: cardano-cli shelley key key-from-mnemonic [--key-output-format STRING]
+                                                   ( --payment-key
+                                                     --payment-address-number WORD32
+                                                   | --stake-key
+                                                     --payment-address-number WORD32
+                                                   | --drep-key
+                                                   | --cc-cold-key
+                                                   | --cc-hot-key
+                                                   )
+                                                   --account-number WORD32
+                                                   ( --mnemonic-from-file FILEPATH
+                                                   | --mnemonic-from-interactive-prompt
+                                                   )
+                                                   --signing-key-file FILEPATH
+
+  Derive an extended signing key from a mnemonic sentence. To ensure the safety
+  of the mnemonic phrase, we recommend that key derivation is performed in an
+  air-gapped environment.
+
+Available options:
+  --key-output-format STRING
+                           Optional key output format. Accepted output formats
+                           are "text-envelope" and "bech32" (default is
+                           "text-envelope").
+  --payment-key            Derive an extended payment key.
+  --payment-address-number WORD32
+                           Payment address number in the derivation path.
+  --stake-key              Derive an extended stake key.
+  --payment-address-number WORD32
+                           Payment address number in the derivation path.
+  --drep-key               Derive an extended DRep key.
+  --cc-cold-key            Derive an extended committee cold key.
+  --cc-hot-key             Derive an extended committee hot key.
+  --account-number WORD32  Account number in the derivation path.
+  --mnemonic-from-file FILEPATH
+                           Input text file with the mnemonic.
+  --mnemonic-from-interactive-prompt
+                           Input the mnemonic through an interactive prompt.
+  --signing-key-file FILEPATH
+                           Output filepath of the signing key.
```

Comment:
```suggestion
  --out-signing-key-file FILEPATH
                           Output filepath of the signing key.
```
I would suggest marking output file so it stands out. It took me a moment to find out the right flag which saves a file.

Reply:
In the end, we decided to stick to with `--signing-key-file` for preserving both consistency and backwards-compatiblity for now